Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM python:3.6
+
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
```

Comment:
TODO: Check if all these dependencies are actually needed, might be able to pare down the image some more